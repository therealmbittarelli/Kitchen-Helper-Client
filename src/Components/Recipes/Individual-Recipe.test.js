import React from 'react';
import ReactDOM from 'react-dom';
import IndividualRecipe from './Individual-Recipe';
import { BrowserRouter } from 'react-router-dom';
import renderer from "react-test-renderer";

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<BrowserRouter>
    <IndividualRecipe />
  </BrowserRouter>, div);
  ReactDOM.unmountComponentAtNode(div);
});

it('renders the UI as expected', () => {
  const tree = renderer.create(<BrowserRouter>
    <IndividualRecipe />
  </BrowserRouter>)
    .toJSON();
  expect(tree).toMatchSnapshot();
});