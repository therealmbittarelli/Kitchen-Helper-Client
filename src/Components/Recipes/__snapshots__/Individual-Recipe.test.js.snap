// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the UI as expected 1`] = `
<div
  className="view"
  id="recipeView"
>
  <div
    className="image-container"
  >
    <div
      className="image"
      style={
        Object {
          "backgroundImage": "url(undefined)",
        }
      }
    />
  </div>
  <p
    className="recipePageHeader"
  >
    Title:
  </p>
  <p
    className="recipeInfo"
  />
  <p
    className="recipePageHeader"
  >
    Recipe Ingredients:
  </p>
  <p
    className="recipeInfo"
  />
  <p
    className="recipePageHeader"
  >
    Recipe Instructions: 
  </p>
  <div
    className="recipeInfo"
  />
  <p
    className="recipePageHeader"
  >
    Time to make the recipe:
  </p>
  <p
    className="recipeInfo"
  >
     Minutes
  </p>
  <div
    className="buttonGroup"
  >
    <div>
      <div
        className="ownerSelectors"
      >
        <a
          className="editRecipe"
          href="/edit-recipe/undefined"
          onClick={[Function]}
        >
          <button
            className="medButton"
          >
            Edit Recipe
          </button>
        </a>
      </div>
    </div>
    <div>
      <div
        className="delete-recipe-button-div"
      >
        <button
          className="medButton"
          onClick={[Function]}
        >
          Delete Recipe
        </button>
      </div>
    </div>
    <a
      href="/recipes"
      onClick={[Function]}
    >
      <button
        className="cancel-view medButton"
      >
        Cancel
      </button>
    </a>
  </div>
</div>
`;
